go.mod

```diff
@@ -9,6 +9,16 @@ require (
 	sigs.k8s.io/controller-tools v0.11.1
 )
 
+require (
```

Comment:
can you combine this and the next `require` blocks?

Reply:
I joined the two indirect blocks into a single one. Now there is one block for the indirect and one for the direct.